deepchem/models/chemberta.py

```diff
@@ -0,0 +1,83 @@
+import torch
```

Comment:
As a second note, could you make a new directory `deepchem/models/torch_models` and move `chemberta.py` into this directory?

Reply:
Sounds good, will do!